qa/evil-tests/src/test/java/org/elasticsearch/plugins/ListPluginsCommandTests.java

```diff
@@ -66,18 +76,150 @@ public void testNoPlugins() throws Exception {
+        assertTrue(output, output.contains("Classname: org.fake"));
+        assertTrue(output, output.contains("Classname: org.fake2"));
+        assertTrue(output, output.contains("Classname: org.fake3"));
+        assertTrue(output, output.contains("fake_plugin"));
+        assertTrue(output, output.contains("fake2_plugin"));
+        assertTrue(output, output.contains("fake3_plugin"));
+        assertTrue(output, output.contains("1.0"));
+        assertTrue(output, output.contains("fake desc"));
+    }
+        
+    public void testPluginWithoutVerboseMultiplePlugins() throws Exception {
+        Environment env = createEnv();
+        
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake1"),
+                "description", "fake desc",
+                "name", "fake_plugin",
+                "version", "1.0",
+                "elasticsearch.version", Version.CURRENT.toString(),
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake");
+
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake2"),
+                "description", "fake desc",
+                "name", "fake2_plugin",
+                "version", "1.0",
+                "elasticsearch.version", Version.CURRENT.toString(),
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake2");
+        
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake3"),
+                "description", "fake desc",
+                "name", "fake3_plugin",
+                "version", "1.0",
+                "elasticsearch.version", Version.CURRENT.toString(),
+                "java.version", System.getProperty("java.specification.version"),
+                "classname", "org.fake3");
+
+        MockTerminal terminal = listPlugins(env, new String[0]);
+        String output = terminal.getOutput();
+        assertFalse(output, output.contains("Plugin information"));
+    }
+    
+    public void testPluginWithoutDescriptionFile() throws Exception{
+        Environment env = createEnv();
+        Files.createDirectories(env.pluginsFile().resolve("fake1"));
+        NoSuchFileException e = expectThrows(NoSuchFileException.class, () -> {
+        	listPlugins(env);
+        });
+        assertTrue(e.getMessage(), e.getMessage().contains("fake1\\plugin-descriptor.properties"));
+    }
+    
+    public void testPluginWithWrongDescriptionFile() throws Exception{
+        Environment env = createEnv();        
+        PluginTestUtil.writeProperties(env.pluginsFile().resolve("fake1"),
+                "description", "fake desc");
+        IllegalArgumentException e = expectThrows(IllegalArgumentException.class, () -> {
+        	listPlugins(env);
+        });
+        assertTrue(e.getMessage(), e.getMessage().contains("Property [name] is missing in"));
+    }
```

Comment:
There should be a newline here between the brace closing the method and the brace closing the class.